ehrql/utils/log_utils.py

```diff
@@ -59,3 +40,8 @@ def init_logging():
             },
         }
     )
+
+
+def kv(kv_pairs):
+    """Generate a string of kv pairs in space separated k=v format."""
+    return " ".join("{}={}".format(k, v) for k, v in kv_pairs.items())
```

Comment:
this is covered by an existing unit test, right?

Reply:
Executed, yes, but it wasn't being checked explicitly, due to us faking the logging in the test.  Have added a commit to use it there.

Reply:
I'd be tempted to stick in a very basic unit test in the existing unit test file as it's such an easy thing to test. Possibly doesn't add much, but I've found having 100% unit test coverage of the utils modules makes them easier to work on by increasing the speed of the feedback loop.